Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -175,7 +175,7 @@ class FontDescription {
     PREFERRED_TYPE(NonCJKGlyphOrientation) unsigned m_nonCJKGlyphOrientation : 1; // Only used by vertical text. Determines the default orientation for non-ideograph glyphs.
     PREFERRED_TYPE(FontWidthVariant) unsigned m_widthVariant : 2;
     PREFERRED_TYPE(TextRenderingMode) unsigned m_textRendering : 2;
-    unsigned m_script : 7; // UScriptCode - Used to help choose an appropriate font for generic font families.
+    PREFERRED_TYPE(UScriptCode) unsigned m_script : 9; // Used to help choose an appropriate font for generic font families.
```

Comment:
I think I'm missing something, why 9 bits and not 8?
Isn't `USCRIPT_CODE_LIMIT` <= 255?

Should we use something like `std::bit_width` to infer the number of bits needed based on `USCRIPT_CODE_LIMIT`?

Reply:
Can `static_assert` right here to make it clear that it's the right number of bits.

I also wonder if there's a test that could demonstrate the problem.

Reply:
> I think I'm missing something, why 9 bits and not 8?
> Isn't `USCRIPT_CODE_LIMIT` <= 255?
> 
> Should we use something like `std::bit_width` to infer the number of bits needed based on `USCRIPT_CODE_LIMIT`?

-1 is also a valid value.